README.md

```diff
@@ -26,6 +27,15 @@ Please check documentation attached to each crate:
 * [**restrict-validity-period**](restrict-validity-period/README.md) restricts
   the validity period of a [NTFS] dataset and purges out-of-date data.
 
+## Setup Rsut environment
+
+`transit_model` is developed in [Rust].
+
+If you want to contribute or install binaries, you need to install a Rust environment.\
+See https://rustup.rs/ for the installation of the [Rust] environment.
```

Comment:
To avoid repetitions:
```suggestion
If you want to contribute or install binaries, you need to install a [Rust] environment: see https://rustup.rs
```